platform/ext/target/nordic_nrf/nrf5340/nrf5340.cmake

```diff
@@ -12,11 +12,42 @@ include("Common/CpuM33")
 
 set(NRF5340_DIR ${CMAKE_CURRENT_LIST_DIR})
 
-include("${NRF5340_DIR}/../common/common.cmake")
+embedded_include_directories(PATH "${NRF5340_DIR}/partition" ABSOLUTE)
```

Comment:
We'll need also "`${NRF5340_DIR}`" in inclusion paths if `target_cfg.h` and `tfm_peripherals_def.h` are to stay separate for nRF5340 and nRF9160.